### ISSUE TITLE
‘’ Either .total_ops ’’ is the error log?

### ISSUE THREAD

**Issue by zihaozhang9** [Lyken17/pytorch-OpCounter#58]
logs:
`
Either .total_ops or .total_params is already defined in Conv2d(3, 16, kernel_size=(3, 3), stride=(2, 2), padding=(1, 1), bias=False).Be careful, it might change your code's beh
avior.
Either .total_ops or .total_params is already defined in BatchNorm2d(16, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True).Be careful, it might change your code's
behavior.
`

**Comment by Lyken17** [comment]
total_ops is already defined in your model, as shown in the log file.

**Comment by ydhongHIT** [comment]
> total_ops is already defined in your model, as shown in the log file.

I also encounter this problem. How to solve it?
